src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml

```diff
@@ -259,6 +259,51 @@ slots:
     range: JgiMtInterface
     inlined: true
     inlined_as_list: true
+  dna_absorb1
```

Comment:
@pkalita-lbl also hoping you can provide guidance here..

In nmdc-schema

```
 dna_absorb1:
    is_a: biomaterial_purity
    description: 260/280 measurement of DNA sample purity
    title: DNA absorbance 260/280
    comments:
      - Recommended value is between 1 and 3.
    examples:
      - value: '2.02'
    rank: 7
    range: float
    slot_group: JGI-Metagenomics
    recommended: true
```
is in core.yaml

`biomaterial_purity`
is defined in attribute_values.yaml

```
biomaterial_purity:
    range: QuantityValue
```

Is that a problem? QuantityValue I mean.
Do I need to add biomaterial_purity here too? Or just add a regex?